build/CoreGraphics/dll/CoreGraphics.def

```diff
@@ -450,6 +450,7 @@ LIBRARY CoreGraphics
 
         ; private exports below
         _CGPatternCreateFromImage
+        _CGPatternCreateColorspace
```

Comment:
actually it was in UIColor but now that's being removed. taking it out.

---
In reply to: [109017192](https://github.com/Microsoft/WinObjC/pull/2374#discussion_r109017192) [](ancestors = 109017192)